distributions/common/src/templates/TemplateContainer.use.js

```diff
@@ -220,14 +220,22 @@ var TemplateContainer = function () {
                 templateNames = templateNames.concat(templateGroups[libraryTemplateGroups[i]]['templates']);
             }
 
+            var uniqueTemplateNames = [];
+            for (var r = 0; r < templateNames.length; r++) {
+                var templateName = templateNames[r];
+                if (uniqueTemplateNames.indexOf(templateName) === -1) {
+                    uniqueTemplateNames.push(templateName);
+                }
+            }
+
```

Comment:
Hvorfor ikke bare 
`var uniqueTemplateNames = new Set(templateNames);
`

Reply:
Fordi `Set` er for nyt i forhold til den version af javascript vi kører i opencat-business. Vi er ikke oppe på ECMA 6 endnu :(